<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Content</class>
 <widget class="QWidget" name="Content">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QScrollBar {
	border: none;
	background: transparent;
}
QScrollBar:vertical {
	width: 12px;
}
QScrollBar:horizontal {
	height: 12px;
}
QScrollBar::handle {
	background: #249dcd;
	border-radius: 5px;
}
QScrollBar::add-page, QScrollBar::sub-page {
	background: transparent;
}
QScrollBar::add-line, QScrollBar::sub-line {
	height: 0;
    width: 0;
}
QScrollArea {
	background: transparent;
}
QLineEdit {
	background: transparent;
	color: black;
 	font-size: 18px;
	font-family: &quot;Arial&quot;;
  	padding: 2px 0px;
	selection-color: white;
	selection-background-color: #249dcd;
	border: none;
}
QLineEdit:enabled {
	background: #003366;
	color: white;
}
#table {
 	background: transparent;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>18</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="buttonsLayout">
     <property name="spacing">
      <number>9</number>
     </property>
     <item>
      <widget class="DesktopButton" name="addButton">
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="DesktopButton" name="deleteButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="DesktopButton" name="editButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Редактировать</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="SearchLine" name="searchLine" native="true">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="WidgetScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="CollectionTable" name="table">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>640</width>
        <height>177</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DesktopButton</class>
   <extends>QPushButton</extends>
   <header>DesktopButton.h</header>
  </customwidget>
  <customwidget>
   <class>SearchLine</class>
   <extends>QWidget</extends>
   <header>widgets/elements/SearchLine.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetScrollArea</class>
   <extends>QScrollArea</extends>
   <header>widgets/WidgetScrollArea.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CollectionTable</class>
   <extends>QWidget</extends>
   <header>widgets/CollectionTable.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
